src/Forms/FixCommonErrors.cs

```diff
@@ -2854,19 +2849,12 @@ private void Fix3PlusLines()
             for (int i = 0; i < Subtitle.Paragraphs.Count; i++)
             {
                 Paragraph p = Subtitle.Paragraphs[i];
-                string text = p.Text;
-
-                if (Utilities.CountTagInText(text, Environment.NewLine) > 1)
+                if (AllowFix(p, fixAction) && Utilities.CountTagInText(p.Text, Environment.NewLine) > 1)

```

Comment:
Check if AllowFix is <code>true</code> before doing any changes which may slow the performance